PYME/LMVis/__init__.py

```diff
@@ -36,4 +36,6 @@
 
 The VisGUI can be launched using the `VisGUI` script.
 
-"""
\ No newline at end of file
+"""
+
+from . import VisGUI as PYMEVisualize
```

Comment:
Is this to make the notebook imports cleaner?

if so, it would probably be better/sufficient to just do the following  `from .VisGUI import ipython_pymevisualize, ipython_visgui`

Having anything import from `VisGUI` in `LMVis.__init__`  is  currently problematic however when we import modules from `LMVis` in non-gui code (prime candidates are `LMVis.pipeline`, `LMVis.visHelpers`, `LMVis.renderers` etc ...) as importing `VisGUI` imports wx (and also does a bunch of other things - e.g. calling  `logging.basicConfig()`) which have the potential to be problematic.

It's a long-standing TODO item to move some of the backend stuff out of LMVis (or conversely move the UI stuff), but has never felt like it's super high priority.